app/src/main/res/layout/fragment_new_pin.xml

```diff
@@ -0,0 +1,110 @@
+        android:backgroundTint="?attr/colorBackgroundFloating"
+        app:layout_constraintTop_toBottomOf="@+id/newpin_top_bar"
+        app:tabIconTint="?attr/colorPrimary"
+        app:tabIndicatorColor="?attr/colorPrimary"
+        app:tabRippleColor="@color/fui_transparent"
+        app:tabTextColor="?attr/colorPrimary" />
+
+    <androidx.viewpager2.widget.ViewPager2
+        android:id="@+id/newpin_view_pager"
+        android:layout_width="match_parent"
+        android:layout_height="350dp"
+        android:backgroundTint="?attr/colorPrimary"
+        app:layout_constraintTop_toBottomOf="@+id/newpin_tab_layout" >
+
+    </androidx.viewpager2.widget.ViewPager2>
+
+    <Button
+        android:id="@+id/drop_pin_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/drop_pin_button_text"
+        android:layout_marginTop="15dp"
+        android:layout_marginLeft="15dp"
+        android:layout_marginRight="15dp"
+        android:layout_marginBottom="85dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/newpin_textinputlayout" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/newpin_textinputlayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
+        app:layout_constraintTop_toBottomOf="@+id/newpin_caption_text">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/newpin_inputeditlayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="15dp"
+            android:layout_marginRight="15dp"
+            android:layout_marginTop="15dp"
+            android:layout_marginBottom="10dp"
+            android:gravity="top|left"
+            android:lines="3"
+            />
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <TextView
+        android:id="@+id/newpin_caption_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="15dp"
+        android:text="@string/newpin_caption_title"
+        android:textSize="20dp"
+        app:layout_constraintTop_toBottomOf="@+id/newpin_view_pager"
+        app:layout_constraintStart_toStartOf="parent"
+        />
+
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="15dp"
+        android:text="$200"
+        android:textSize="24sp"
+        app:layout_constraintBottom_toBottomOf="@+id/drop_pin_button"
+        app:layout_constraintEnd_toStartOf="@id/drop_pin_button"
+        app:layout_constraintTop_toTopOf="@+id/drop_pin_button" />
```

Comment:
Were you still planning on making this part of the button? I think it'd look a good amount better.